argoverse/evaluation/detection/utils.py

```diff
@@ -338,10 +338,7 @@ def rank(dts: np.ndarray) -> Tuple[np.ndarray, np.ndarray]:
     ranked_scores = scores[ranks]
 
     # Ensure the number of boxes considered per class is at most `MAX_NUM_BOXES`.
-    if ranked_dts.shape[0] > MAX_NUM_BOXES:
```

Comment:
Can you change the base of this PR to point to the branch used in 227 as I am sure this change is not part of this PR?